components/typography/Monospaced.js

```diff
@@ -0,0 +1,42 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import s from './theme.css';
+
+const factory = (defaultElement) => {
+  class Monospaced extends PureComponent {
+    static propTypes = {
+      children: PropTypes.node,
+      className: PropTypes.string,
+      element: PropTypes.node,
+    };
+
+    static defaultProps = {
+      className: '',
```

Comment:
Don't think you need to specify this empty string as default prop, undefined will also work, so you can remove it. (But check it first!)